analyzer/src/funTest/kotlin/managers/GradleAndroidFunTest.kt

```diff
@@ -77,6 +77,20 @@ class GradleAndroidFunTest : StringSpec() {
 
             result.toYaml() shouldBe expectedResult
         }
+
+        "Cyclic dependencies can be handled" {
```

Comment:
Could you highlight the semantic differnece to `"deal with cycles in dependencies"` in `DependencyGraphBuilderTest`?

Reply:
I have improved the test name. The difference is basically that the test in `DependencyGraphBuilder` uses a minimum cycle. In this real-world example, the cycle spans multiple packages; this causes a different behavior of the builder.